Automattic/jetpack
# PHPCS: Comments

Second of two PRs that closes #21960

The remaining fix is at this PR: https://github.com/Automattic/jetpack/pull/22770

#### Changes proposed in this Pull Request:

* This PR updates the comments module to meet PHPCS standards:
  * "projects/plugins/jetpack/modules/comments/admin.php" 
  * "projects/plugins/jetpack/modules/comments/base.php" 
  * "projects/plugins/jetpack/modules/comments/comments.php"


#### Jetpack product discussion

n/a

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* Try testing changing the comments settings (Settings > Discussion)
* Try testing adding a comment section to a test site
* Whilst doing the above, check for any broken experiences or PHPCS errors
